packages/site/src/features/update-snap/Update.tsx

```diff
@@ -4,8 +4,8 @@ import { Result, Snap } from '../../components';
 import { useGetSnapsQuery, useInstallSnapMutation } from '../../api';
 
 const UPDATE_SNAP_ID = 'npm:@metamask/test-snap-bip32';
-const UPDATE_SNAP_OLD_VERSION = '4.6.3';
-const UPDATE_SNAP_NEW_VERSION = '4.6.4';
+const UPDATE_SNAP_OLD_VERSION = '5.0.0';
+const UPDATE_SNAP_NEW_VERSION = '5.1.0';
```

Comment:
```suggestion
const UPDATE_SNAP_NEW_VERSION = '5.0.1';
```
I think its fine for the next version to be a patch release